frontend/src/Pages/MembersOverview.tsx

```diff
@@ -15,6 +15,8 @@ import AccountBoxIcon from '@mui/icons-material/AccountBox';
 export default function MembersOverview() {
     const [modal, setModal] = useState(false);
     const [membersDB, setMembersDB] = useState<Member[] | null>(null);
+    const [search, setSearch] = useState<string>('');
+    console.log(searchMembers());
 
     function fetchMembers(): void {
         axios.get('api/members')
```

Comment:
usually requires / before api, but if you haven't run into any issues, then obv doesn't matter
```suggestion
        axios.get('/api/members')
```